packages/schema/src/config/server.ts

```diff
@@ -9,8 +9,8 @@ export default {
    * export default {
    *   server: {
    *     https: {
-   *       key: fs.readFileSync(fileURLToPath(new URL('./server.key', import.meta.url))),
-   *       cert: fs.readFileSync(fileURLToPath(new URL('./server.crt', import.meta.url)))
+   *       key: fs.readFileSync(fileURLToPath(new URL('./server.key', import.meta.url)),
+   *       cert: fs.readFileSync(fileURLToPath(new URL('./server.crt', import.meta.url))
```

Comment:
I think these are actually correct:
```suggestion
   *       key: fs.readFileSync(fileURLToPath(new URL('./server.key', import.meta.url))),
   *       cert: fs.readFileSync(fileURLToPath(new URL('./server.crt', import.meta.url)))
```

Reply:
You're right. I got something wrong when I changed it.